osbs/build/build_request.py

```diff
@@ -1230,6 +1230,12 @@ def render_group_manifests(self):
         if not self.dj.dock_json_has_plugin_conf('postbuild_plugins', 'group_manifests'):
             return
 
+        if self.spec.flatpak.value:
+            logger.info("removing group_manifests from request "
+                        "because this is a Flatpak built from a module")
+            self.dj.remove_plugin('postbuild_plugins', 'group_manifests')
```

Comment:
I don't think we want to remove the plugin altogether. Wouldn't we instead want to ensure that its "group" parameter is set to False?

Better still would be to raise an OsbsValidationException if `flatpak` is set for the build and `group_manifests` is set for the instance.

Reply:
It was failing for me when it was set to `False`. There is not reason to have it here anyway:
* `group=True` doesn't make sense for flatpaks - these don't use manifest lists (flatpak automatically splits arch-specific image in a branch and pulls the branch based on client's arch
* `group=False` is not vital for flatpak workflow - currently only koji archives are used as a source.

This, however, is a temporary solution, a coordination with Fedora folks is required to proceed, as its not yet clear if OCI registry would be used as a source in future

Reply:
Let's add a unit test for this.